frontend/src/services/userServices.ts

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+
+const API_URL = `http://10.10.0.117:8080/users/register`;
+
+const emailConfirmation = (token?: string) =>
```

Comment:
I think the token param should not be optional here

Reply:
Okey then is it ok to put string or undefined?